src/docker-in-docker/install.sh

```diff
@@ -10,6 +10,7 @@
 
 DOCKER_VERSION="${VERSION:-"latest"}" # The Docker/Moby Engine + CLI should match in version
 USE_MOBY="${MOBY:-"true"}"
+MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION:-"latest"}"
```

Comment:
I think it's alright to formalize this Feature option, can you add an entry to https://github.com/devcontainers/features/blob/main/src/docker-in-docker/devcontainer-feature.json ?

Reply:
Let's set the `default` value to whichever version currently works. This way folks would have a smooth experience. I'd make sure to remove on Monday or whenever Microsoft fixes it.

Also, add a comment on why set the `default` to a specific version.

Reply:
> This patch will still require that consumers update their devcontainer.json files, so perhaps temporarily pinning would be a better immediate fix?

Yep!